spec/honeycomb/configuration_spec.rb

```diff
@@ -2,15 +2,12 @@
 
 RSpec.describe Honeycomb::Configuration do
   let(:configuration) { Honeycomb::Configuration.new }
-  let(:dataset_name) { "dataset" }
-  let(:service_name) { "service_name" }
-  let(:write_key) { "service_name" }
+  let(:write_key) { "d68f9ed1e96432ac1a3380" }
```

Comment:
Is this a made-up key?

Reply:
AFAIK it was randomly generated by @MikeGoldsmith ... I believe I asked the same question 😅